lib/features/mailbox_dashboard/data/network/spam_report_api.dart

```diff
@@ -35,8 +38,12 @@ class SpamReportApi {
           ReferencePath.idsPath,
         ));
     final getMailboxInvocation = requestBuilder.invocation(getMailBoxMethod);
+
+    final capabilities = getMailBoxMethod.requiredCapabilities.toList()
```

Comment:
why we need to convert `toList` then `toSet`? we can do better.

Reply:
U can write an `extension` for Set<Capability>